src/components/manager/ManagerDropBoxContent.js

```diff
@@ -39,6 +37,7 @@ import TableSelectionModal from "./TableSelectionModal";
 import {STEP_INPUT} from "./workflowCommon";
 import {withBasePath} from "../../utils/url";
 import {dropBoxTreeStateToPropsMixinPropTypes, workflowsStateToPropsMixin} from "../../propTypes";
+import { JsonArrayDisplay, JsonObjectDisplay } from "../JsonDisplay";
```

Comment:
move this up to near table selection modal to be grouped with the other component import